package.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "wabur",
+    "version": "0.2.0",
+    "scripts": {
+        "build": "babel -o view/pages/assets/js/wab.js view/js/wab.js view/js/list.js view/js/obj.js view/js/view.js"
```

Comment:
haven't tried this syntax yet.. but there's an `--out-dir` switch and `-w` to compile an entire directory and watch it for any changes..
perhaps the order is necessary to compile the JS in that order?

Reply:
The order was necessary to get it to work. wab.js had to be first which was not the default.